src/qibo/ui/mpldrawer.py

```diff
@@ -629,18 +632,16 @@ def _plot_params(style: Union[dict, str, None]) -> dict:
     return style
 
 
-def plot_circuit(circuit, scale=0.6, cluster_gates=True, style=None):
+def plot_circuit(circuit, scale=0.6, cluster_gates=True, style=None, dpi=100):
```

Comment:
As it is now, the `PLOT_PARAMS` dictionary is collecting parameters useful in very different functions/calls of the plotter, putting together arguments of very different and unrelated objects. 

In this case, for example, the `dpi` argument can be passed to `plt.figure`, to the `plt.savefig` function or set directly into the `plt.rcParams`. 

We could add it as a new argument of the `PLOT_PARAMS` and this is already better than adding an extra argument to the `plot_circuit` function indeed. However, I feel a bit confusing to manage all these (different) tasks using the same dictionary. In particular because for now it is conceived as a tool to customize the style (looking at the example).

Reply:
Well, you already added `dpi` to `PLOT_PARAMS` before my comment:
https://github.com/qiboteam/qibo/blob/08cf82f5374d604fadfbd9a8c377f3c1d28056ed/src/qibo/ui/mpldrawer.py#L28

In any case, there is a single task involved: `plt.rcParams`. Ttbomu, `PLOT_PARAMS` is just a subset of that, with some defaults set (and then, you have various sets of overrides to pick a certain style). Everything else is just a consequence.